src/frame/endpoints/tests/GroupApiEndpoints.test.ts

```diff
@@ -184,16 +191,20 @@ describe("GroupApiEndpoints", () => {
             );
         });
 
-        it("checks contents of transformKey, if undefined payload should not have transformKey key or member list", () => {
+        it("checks contents of transformKeyGrantList, if empty payload should not have member list", () => {
+            const groupPublicKey = {
+                x: new Uint8Array([98, 105, 133]),
+                y: new Uint8Array([110, 98]),
+            };
             const groupEncryptedPrivateKey: any = {
                 encryptedMessage: "abc",
                 authorizationCode: "auth",
                 ephemeralPublicKey: "epub",
             };
 
-            const transformKey = undefined;
+            // const transformKeyGrantList = [];
 
-            GroupApiEndpoints.callGroupCreateApi("", groupPublicKey, groupEncryptedPrivateKey, false, "group name", transformKey).engage(
+            GroupApiEndpoints.callGroupCreateApi("", groupPublicKey, groupEncryptedPrivateKey, false, [], "group name").engage(
```

Comment:
It'd probably be good to add a test where we pass in two transform keys to make sure that the request that is sent includes them both.

Reply:
I added this check to the test "combines all group content into payload and maps response to data result".